src/modules/netinstall/NetInstallPage.cpp

```diff
@@ -48,13 +48,18 @@
 
 using CalamaresUtils::yamlToVariant;
 
+
+const char* NetInstallPage::HEADER_TEXT = "<h1>Package Selection</h1>";
```

Comment:
- Translations don't work that way. You need to keep the literal (English) string inside the tr() call.
 - It would be `static const char HEADER_TEXT[]=` anyway, because there's no need to expose it in the class

Reply:
it is static and private, defined in the .h - where would you prefer such constants to be? especially if they pretain to only this module